monai/transforms/inverse.py

```diff
@@ -71,7 +71,7 @@ def push_transform(
             info[TraceKeys.DO_TRANSFORM] = self._do_transform  # type: ignore
 
         if key in data and isinstance(data[key], MetaTensor):
-            data[key].push_transform(info)
+            data[key].push_applied_operation(info)
```

Comment:
I also thought this is not correct when I was reviewing, but I think your unit tests cover it.

Thanks.